source/StigData/Archive/Windows.Server.2022/U_MS_Windows_Server_2022_MS_STIG_V1R1_Manual-xccdf.log

```diff
@@ -0,0 +1,19 @@
+V-254248::*::HardCodedRule(ServiceRule)@{DscResource = 'Service'; Ensure = 'Present'; ServiceName = $null; ServiceState = 'Running'; StartupType = $null; OrganizationValueTestString = 'ServiceName/StartupType is populated with correct AntiVirus service information'}
+V-254255::*::''
+V-254265::*::HardCodedRule(ServiceRule)@{DscResource = 'Service'; Ensure = 'Present'; ServiceName = $null; ServiceState = 'Running'; StartupType = $null; OrganizationValueTestString = 'ServiceName/StartupType is populated with correct Firewall service information'}
+V-254291::"Minimum password length,"::"Minimum password length"
+V-254356::0x00000000 (0) (Security), 0x00000001 (1) (Basic)::0 or 1
+V-254357::0x00000000 (0) - No peering (HTTP Only)::0, 1, 2, 99 or 100
+V-254362::0x00000000 (0) (or if the Value Name does not exist)::0
+V-254363::0x00000000 (0) (or if the Value Name does not exist)::0
+V-254364::0x00000000 (0) (or if the Value Name does not exist)::0
+V-254371::0x00000000 (0) (or if the Value Name does not exist)::0
+V-254375::0x00000000 (0) (or if the Value Name does not exist)::0
+V-254443::DoD Root CA 3- DoD Interoperability Root CA 2 - 49CBE933151872E17C8EAE7F0ABA97FB610F6477::DoD Root CA 3 - DoD Interoperability Root CA 2 - 49CBE933151872E17C8EAE7F0ABA97FB610F6477
+V-254443::Subject: CN=DoD Root CA 2, OU=PKI, OU=DoD, O=U.S. Government, C=US::Subject: CN=DoD Root CA 3, OU=PKI, OU=DoD, O=U.S. Government, C=US
+V-254443::Thumbprint: A8C27332CCB4CA49554CE55D34062A7DD2850C02::Thumbprint: 49CBE933151872E17C8EAE7F0ABA97FB610F6477
+V-254443::NotAfter: 8/26/2022 9:25:51 AM::NotAfter: 11/16/2024
+V-254458::*::HardCodedRule(RegistryRule)@{DscResource = 'Registry'; Ensure = 'Present'; Key = 'HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System'; ValueName = 'LegalNoticeCaption'; ValueType = 'String'; ValueData = $null; OrganizationValueTestString = "'{0}' -match '^(DoD Notice and Consent Banner|US Department of Defense Warning Statement)$'"}
+V-254484::0x00000002 (2) (Prompt for consent on the secure desktop)::1 or 2
+V-254490::0x00000002 (2) (or if the Value Name does not exist)::2
```

Comment:
What is the logic behind removing this? PowerSTIGs composite will take 'shouldBEAbsent' and mark $ensure as absent.. which I believe the STIG is saying.. either 2 or doesn't exist? 
<img width="629" alt="image" src="https://user-images.githubusercontent.com/44269812/224127825-8fe0f5ab-eda3-4dd3-b82b-cc0b10ddf601.png">

Reply:
@erjenkin Not following the code snippet of the .log file above vs. question of removal of should be absent.  If you are talking about the org settings file and removing the line of the setting, that line is not generated from the CreateOrgSettingsFile.

Reply:
Sorry not being clear.

You created a log file entry :
V-254490::0x00000002 (2) (or if the Value Name does not exist)::2
PowerSTIG has logic to test for 'of if the Value Name does not exist'
<img width="584" alt="image" src="https://user-images.githubusercontent.com/44269812/224132139-ae445847-5e89-41ad-ba7b-eccb67b32e65.png">

Your log file entry changes this from an organization setting that consumers can use to either set this value to 2 or ShouldBeAbsent
Can you just remove this log file entry and let it convert as an organization setting?

Reply:
@erjenkin Without out it, it parses as:

<img width="940" alt="image" src="https://user-images.githubusercontent.com/16927960/224135494-73f6b6f4-be23-46a9-95ed-66ae220976e4.png">

Reply:
Looks like there are a few more in your log file also - 
V-254362::0x00000000 (0) (or if the Value Name does not exist)::0
V-254363::0x00000000 (0) (or if the Value Name does not exist)::0
V-254364::0x00000000 (0) (or if the Value Name does not exist)::0
V-254371::0x00000000 (0) (or if the Value Name does not exist)::0
V-254375::0x00000000 (0) (or if the Value Name does not exist)::0

Reply:
Being handled the exact same way as V-205924 in Windows Server 2019

Reply:
All being handled the exact same way as they were handled in parsing the Windows Server 2019 STIGs.

Reply:
Hum tough to figure out logic for the old ones, but works for me, sorry for the back and forth :)